application/client/src/app/ui/service/toolbar.ts

```diff
@@ -7,13 +7,15 @@ import { unique } from '@platform/env/sequence';
 import { TabsService, ITab } from '@elements/tabs/service';
 import { UUIDs } from './toolbar/register';
 import { Tabs } from './toolbar/tabs';
+import { Chart } from './toolbar/chart';
 
 @SetupService(ui['toolbar'])
 export class Service extends Implementation {
     private _emitter!: Emitter;
     private _channel!: Channel;
     private _services!: Services;
     private _available!: Tabs;
+    private _chart!: Chart;
```

Comment:
Chart service should not be a part of the toolbar service. It should "live" in the scope of the chart view.